python/django_bolt/_worker_loop.py

```
@@ -890,11 +285,10 @@ def mcp_resolve_future(fut, value):
         fut.set_result(value)
 
 
-def run_worker_handle(loop, handle):
-    """Execute one ready Handle with the WorkerLoop installed as running."""
-    if handle.cancelled():
-        return
-    try:
-        _with_running_loop(loop, handle._run)
-    except BaseException as exc:
-        loop.call_exception_handler({"message": "Exception in WorkerLoop callback", "exception": exc, "handle": handle})
+def worker_handle_failed(loop, handle, exc):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `worker_handle_failed`

Add return type annotation: `None`

(ANN202)